ibm/data_source_ibm_tg_gateway.go

```diff
@@ -171,6 +171,10 @@ func dataSourceIBMTransitGatewayConnectionsRead(d *schema.ResourceData, meta int
 		if instance.NetworkID != nil {
 			tgConn[tgNetworkId] = *instance.NetworkID
 		}
+		if instance.NetworkAccountID != nil {
+			tgConn[tgNetworkAccountID] = *instance.NetworkAccountID
```

Comment:
Did you defined the tgNetworkAccountID in the schema?

Reply:
Added tgNetworkAccountID in schema